src/sentry/models/rule.py

```diff
@@ -73,7 +67,10 @@ class Rule(Model):
     class Meta:
         db_table = "sentry_rule"
         app_label = "sentry"
-        indexes = (models.Index(fields=("project", "status", "owner")),)
+        indexes = (
+            models.Index(fields=("project", "status", "owner_team")),
+            models.Index(fields=("project", "status", "owner_user_id")),
+        )
         constraints = (
```

Comment:
Hmm, I *think* we'll be ok without index coverage here for a short time. Since we'll recreate the `project, status, *` index pretty fast, and I don't know that the `owner` part actually makes a huge different

Reply:
There are very few queries for `Rule` that use the owner fields. The most frequently used one is likely in the combined_rules endpoint where a team condition is added if ownership filtering is performed.